cmd/prepare-storage.go

```diff
@@ -280,7 +275,7 @@ func retryFormattingXLDisks(firstDisk bool, endpoints EndpointList, storageDisks
 					printRegularMsg(endpoints, storageDisks, printOnceFn())
 				}
 				return err
-			case WaitForHeal:
+			case SuggestToHeal:
```

Comment:
The message printed in `case WaitForFormatting` should be "Initializing data volume for first time. Waiting for first server to complete formatting disks (elapsed %s)\n". This is not related to PR but found while reviewing.